README.md

```diff
@@ -148,7 +148,7 @@ If you get stuck or just want advice, come chat with the engineers on [Discord][
 [latest-version]: https://github.com/secrethub/secrethub-go/releases/latest
 [issues]: https://github.com/secrethub/secrethub-go/issues/new
 [pulls]: https://github.com/secrethub/secrethub-go/pulls
-[godoc]: http://godoc.org/github.com/secrethub/secrethub-go
+[godoc]: https://pkg.go.dev/github.com/secrethub/secrethub-go?tab=overview
```

Comment:
@Marton6 is it intentional that this points to the `overview` tab? If it is fine, if not, please remove the query params.

Reply:
Ah nevermind, just saw you had a conversation about this specifically. Resolving my comment...